### PR TITLE
Add Mobile Jazz and Bugfender

### PR DESCRIPTION
**Pull request by adchsm**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @adchsm, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 2400c83).  See below for details:

```
README.md: Company is listed out of order: "Mobile Jazz" (should be before "Mokriya")
mobile-jazz.md: Profile looks complete, but the main readme contains a warning emoji.

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @adchsm, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit fa51598).  See below for details:

```
mobile-jazz.md: Profile looks complete, but the main readme contains a warning emoji.

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

File: company-profiles/bugfender.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Worldwide
+
+## Office locations
+
+None
+
+## How to apply
+
+You can find our open jobs and apply via our API here: https://mobilejazz.com/jobs
```
[the remark this answers is not in the document]
Comment (adchsm) [reply company-profiles/bugfender.md]: Hey James, that's actually correct but I see where the confusion comes from. Bugfender is a sub-team and product from Mobile Jazz, but we manage all hiring through MJ's site & API. Hope that helps to clear things up.

[the remark this answers is not in the document]
Comment (nylen) [reply company-profiles/bugfender.md]: I see. Ok, merging this then. Thanks for the PR!

**Pull request merged**